src/sdk/modules/utils/Types.ts

```diff
@@ -82,58 +19,7 @@ export type CreateSessionDataResponse = {
   sessionIDInfo: Array<string>
 }
 
-export interface CreateSessionDataParams {
-  /** window end for the session key */
-  validUntil: number
-  /** window start for the session key */
-  validAfter: number
-  /** Address of the session validation module */
-  sessionValidationModule: Hex
-  /** Public key of the session */
-  sessionPublicKey: Hex
-  /** The hex of the rules {@link Rule} that make up the policy */
-  sessionKeyData: Hex
-  /** we generate uuid based sessionId. but if you prefer to track it on your side and attach custom session identifier this can be passed */
-  preferredSessionId?: string
-}
-
-export interface MultiChainValidationModuleConfig
-  extends BaseValidationModuleConfig {
-  /** Address of the module */
-  moduleAddress?: Hex
-  /** Version of the module */
-  version?: ModuleVersion
-  /** Signer: viemWallet or ethers signer. Ingested when passed into smartAccount */
-  signer: UnknownSigner
-}
-export interface MultiChainValidationModuleConfigConstructorProps
-  extends BaseValidationModuleConfig {
-  /** Address of the module */
-  moduleAddress?: Hex
-  /** Version of the module */
-  version?: ModuleVersion
-  /** Signer: viemWallet or ethers signer. Ingested when passed into smartAccount */
-  signer: Signer
-}
-
-export interface BaseSessionKeyData {
-  sessionKey: Hex
-}
-
-export interface ERC20SessionKeyData extends BaseSessionKeyData {
-  /** ERC20 token address */
-  token: Hex
-  /** Recipient address */
-  recipient: Hex
-  /** ERC20 amount (Bigint) */
-  maxAmount: bigint
-}
-
-export interface SessionValidationModuleConfig {
-  /** Address of the module */
-  moduleAddress: string
-}
-
+// Review: if needed
 export type V3ModuleInfo = {
```

Comment:
Isn't this same as "Module" type from module-sdk ?

Reply:
yeah it should not be needed